docs/user-guide/tls.md

```diff
@@ -10,7 +10,7 @@ Anytime we reference a TLS secret, we mean a PEM-encoded X.509, RSA (2048) secre
 You can generate a self-signed certificate and private key with:
 
 ```bash
-$ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ${KEY_FILE} -out ${CERT_FILE} -subj "/CN=${HOST}/O=${HOST}"
+$ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout xpto.key -out xpto.crt -subj "/CN=xpto.com/O=xpto.com" -addext "subjectAltName = DNS:xpto.com"
```

Comment:
@longwuyuan replace xpto.com for ${HOST} maybe? :)

Reply:
Doing now :-)

Reply:
Done. Please check @rikatz